packages/build-utils/src/types.ts

```diff
@@ -597,6 +597,12 @@ export interface Chain {
    * The headers to send when making the request to append to the response.
    */
   headers: Record<string, string>;
+
+  /**
+   * This is used to filter the query parameters that are added from the named
+   * regular expression matches.
+   */
+  filterRegexMatches: string[];
```

Comment:
Why is this needed separate of allowQuery?

Reply:
Shouldn't the resume allow all query values/matches?

Reply:
Not all the ones from the regular expression named matches. This is the source of the `?rscstuff=.rsc` query parameter that was being added by some regular expressions.